### ISSUE TITLE
Source of language datasets

### ISSUE THREAD

**Issue by DonaldTsang** [richtr/guessLanguage.js#14]
Where is the source text dataset for the Ngrams of those 100 languages? Would like to see if it is different from wooorm/franc#78 usage of UDHR, and if it is more accurate than them.

**Comment by Animenosekai** [comment]
>Where is the source text dataset for the Ngrams of those 100 languages? Would like to see if it is different from wooorm/franc#78 usage of UDHR, and if it is more accurate than them.

@DonaldTsang I really don’t know because I’m not the dev but isn’t it in `_languageData.js`?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Animenosekai** [comment]
>Where is the source text dataset for the Ngrams of those 100 languages? Would like to see if it is different from wooorm/franc#78 usage of UDHR, and if it is more accurate than them.

@DonaldTsang (inside the lib folder)

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Animenosekai** [comment]
>Where is the source text dataset for the Ngrams of those 100 languages? Would like to see if it is different from wooorm/franc#78 usage of UDHR, and if it is more accurate than them.

@DonaldTsang But it’s weird because there isn’t all language and the ones which are in it are not written in the actual language (for example: in “fr” it isn’t written in French and I don’t understand what’s written)

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Animenosekai** [comment]
>Where is the source text dataset for the Ngrams of those 100 languages? Would like to see if it is different from wooorm/franc#78 usage of UDHR, and if it is more accurate than them.

@DonaldTsang The dev used primarily Unicode checking to determine the language tho

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by DonaldTsang** [comment]
@Animenosekai if it does only use Unicode checking, that would actually be really sweet as that is very useful for my cause of making language checking easier (which I hope can re implement in Python).

**Comment by DonaldTsang** [comment]
The ` _languageData.js` seems like N-Gram data.

**Comment by Animenosekai** [comment]
> @Animenosekai if it does only use Unicode checking, that would actually be really sweet as that is very useful for my cause of making language checking easier (which I hope can re implement in Python).

I don't think that it uses only Unicode checking but why don't you open `guessLanguage.js` as it should contain everything you wanna know
